proxy/src/transport/tls/config.rs

```diff
@@ -249,15 +249,27 @@ impl CommonConfig {
 
 }
 
+// A Future that, when polled, checks for config updates and publishes them.
+pub type PublishConfigs = Box<Future<Item = (), Error = ()> + Send>;
```

Comment:
Thanks for naming this --- I think it was originally unnamed because this returned `impl Future`, but as it's now boxed, it should definitely have a named type!